cmd/govim/testdata/scenario_default/workspace_mode.txt

```diff
@@ -0,0 +1,96 @@
+# Test workspace mode where a go.work file exist in the parent directory
```

Comment:
nit: use periods to mark the end of sentences, otherwise I first read this as if the second line was a continuation.